### DISCUSSION TITLE
Any toy examples showing sphinx-needs in action?

### DISCUSSION THREAD

**Discussion by melund** [useblocks/sphinx-needs#1060] [Q&A] [answered]
I wanted to show management some examples of sphinx needs in action. 

Are there any small toy-examples that I could clone and use? Or any open-source uses of sphinx-needs that would be suitable for the same task?

**Comment by danwos** [comment, marked as answer]
I'm sorry, but there are no simple example projects of Sphinx-Needs available right now.
That's a common request, but I haven't found the time to create such examples.

All I can give you are the official Sphinx-Needs docs and the hint for doing a github search, to find repos which have `sphinx_needs` in their `conf.py` file, maybe there are some helpful repos: https://github.com/search?q=path%3Aconf.py+%22sphinx_needs%22&type=code&p=1

However, we have some company internal docs using Sphinxy-Needs and experiences from customer projects, which I could share in a private session. Just get in contact with me via daniel.woste@useblocks.com.

**Reply by melund** [reply]
Thanks a lot @danwos. I will look on GitHub and get back to you if it becomes relevant.
